installer/dev/WindowsAppRuntimeInstall.rc

```diff
@@ -6,45 +6,45 @@
 //
 // PACKAGE
 //
-#include "windowsappsdk_definitions.h"
-#if defined(WAS_FRAMEWORK_X86_RCENTRY)
-WAS_FRAMEWORK_X86_RCENTRY
+#include "windowsappruntime_definitions.h"
+#if defined(WAR_FRAMEWORK_X86_RCENTRY)
+WAR_FRAMEWORK_X86_RCENTRY
```

Comment:
Unfortunate acronym. Maybe this is `WINAPPRUNTIME_` ?

Reply:
Would prefer it be left `WAR`. There's no confusion.

Reply:
> Unfortunate acronym. Maybe this is `WINAPPRUNTIME_` ?

Even more unfortuntely the ResourceCompiler's preprocessor is archaic. These are used by WindowsAppRuntimeInstall.rc and, unlike C/C++ compilers and MIDL, RC.EXE's preprocessor appears to be firmly wedged in 1985 including errors if #defines are >32 characters long. @dkbennett originally used full names and hit length issues, which lean to shorting `PROJECTREUNION_*` to `PR_*` and then the renaming fun to `WAS_*` and now `WAR_*`.

We almost always used the full form "Windows App SDK" or the shortened "WinAppSDK". The only time we resort to the abbreviation is under extreme necessity. To date the only places I'm aware of are here, [/specs/Deployment/MSIXPackageVersioning.md](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/Deployment/MSIXPackageVersioning.md) and the [Windows App SDK Deployment API Design spec #1253](https://github.com/microsoft/WindowsAppSDK/pull/1253). The .rc due to RC.EXE limitations, and in those 2 docs because of the heavy terminology reference where using the full names would degrade clarity.